### DISCUSSION TITLE
Allow access to LNURL payment’s comment field through the API

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#3025] [General]
As discussed on the dev call with @Kukks

**Comment by bolatovumar** [comment]
Does anyone know how I can actually test LNURL payment comments in development? I can probably do this but I have no idea how I would actually test it.

**Comment by woutersamaey** [comment]
@pavlenex or @Kukks should we move this to an issue instead of a discussion? It's a task we should not forget for API completeness sake.

**Reply by pavlenex** [reply]
The context to what's proposed is unclear, but moving it over.

**Reply by woutersamaey** [reply]
If I understand correctly: With LNURL payments you now have a `comments` field that goes along with the payment. Users can type in whatever they want. This is new and is not in Greenfield yet.
As you know BTC transactions don't have a comments field, but LNURL payments do.

**Reply by pavlenex** [reply]
Awesome, makes sense.

**Reply by woutersamaey** [reply]
Did you manage to move this to an issue?

**Reply by bolatovumar** [reply]
@woutersamaey yeah, there is an issue for it here: https://github.com/btcpayserver/btcpayserver/issues/3154
